clients/ptranslator/README.adoc

```diff
@@ -70,41 +66,54 @@ udp_send_channel {
 === StatsD
 
 The converter accepts statsD packets sent via UDP (to the UDP port).
-Format is
+Format is:
+----
+key:value|type
+----
 
-  key:value|type
+Where only key and value are accepted and the type is ignored (for now).
 
+Example command:
+----
+$ echo "my_time:100|ms" | nc -u4 localhost 8125
+----
 
-Where only key and value are accepted and the type is ignored (for now).
+=== collectd
 
-E.g.: `$ echo "my_time:100|ms" | nc -u4 localhost 8125`
+You must configure the collectd network plugin to send UDP packets.
+----
+LoadPlugin network
+<Plugin network>
+        <Server "127.0.0.1" "25826">
+        </Server>
+</Plugin>
+----
 
+== Running ptrans for development
 
-== Running ptrans
+Don't forget to start an Hawkular Metrics web server as well.
 
-*With `start.sh`*
+=== With the start script
 
 You can use the provided `start.sh` script to start the server (this will build the source first).
 
-Before you start the proxy, make sure the rest-server is running.
-
 Make sure to specify a configuration file on the command line:
-
+----
  $ ./start.sh -c ptrans.conf
+----
 
-*With Maven*
-
-Once the project is built, simply run the following command:
+This method does not allow to configure logging and works on *nix systems only.
 
- mvn exec:java
+=== With Maven ===
 
-By default, the ptrans.conf configuration at the project base directory will be used.
+Once the project is built, simply run the following command:
+----
+mvn exec:java
+----
 
-== Room for improvement:
+By default, the log4j and ptrans.conf files at the project base directory will be used.
+If you want to use your own files:
+----
+mvn exec:java -Ddev.log4j.configuration=<path to log4j file> -Ddev.ptrans.conf=<path to ptrans.conf>
+----
 
-* add information about the remote to the key so make the key unique
```

Comment:
Why are you removing those? Do we have enhancement jiras instead?  Perhaps link to them?